build.gradle

```diff
@@ -212,6 +212,17 @@ subprojects {
 
 
   ext {
+    applyJetBrainsPlugin = {withGit4ide ->
```

Comment:
Nit: missing space after `{`

Reply:
Nice, a helper like that was missing :)

Reply:
Actually, a more idiomatic Groovy would be sth like: `applyJetBrainsPlugin = { params ->`, where `params` is a Map; then you'd need to do `if (params.withGit4Idea) {` below and `applyJetBrainsPlugin(withGit4Idea: true)`

Reply:
https://mrhaki.blogspot.com/2015/09/groovy-goodness-turn-method-parameters.html

Not sure though if this syntax works with those `ext`'s lambdas, maybe only with regular methods

Reply:
Nice! Applied